### ISSUE TITLE
Sorting when in a category view

### ISSUE THREAD

**Issue by timscaffidi** [atduskgreg/ofxaddons.com#121]
I was looking for a way to find the most popular or freshest addons in a specific category. For example the most popular GUI addons. I did not see a way to do this but I may be missing something.

**Comment by jamezilla** [comment]
You're right this isn't supported at the moment. I'd really like to move this kind of logic into the client, but haven't had much time to work on the site lately. Thanks for mentioning it, though. It's good to get some feedback on what people want/need.
